includes/namespace.php

```diff
@@ -109,13 +109,20 @@ function new_site( $site ) {
 	if ( ! is_multisite() ) {
 		return;
 	}
+
```

Comment:
Lots of added new lines in this diff, why?

Reply:
Figured the code should have some more air to breathe while I was at it.